components/breadcrumbs/breadcrumb.js

```diff
@@ -0,0 +1,93 @@
+import '../icons/icon.js';
+import '../link/link.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class BreadCrumb extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			compact: {
```

Comment:
It might be worth making this private. Its technically accessed by `d2l-breadcrumbs` so its not truly private but we also don't want others setting it directly. Not sure if anyone else has any thoughts here.

Reply:
I was going to suggest the same -- `_compact`.

Reply:
@AllanKerr @dlockhart considering it should be an attribute as well. Should I name the attribute as `compact` still?

Reply:
If we're making it private I would recommend adding it to the `d2l-breadcrumb-wrapper` with `?data-compact=${this.compact}` and using `.d2l-breadcrumb-wrapper[data-compact]` instead. That way we don't need to reflect it to the host anymore.

Reply:
Done ✔️